app/logic/Mail/JSON/JSONMailAccount.ts

```diff
@@ -39,7 +39,7 @@ export class JSONMailAccount {
     acc.userRealname = sanitize.label(json.userRealname, appGlobal.me.name);
     acc.name = sanitize.label(json.name, acc.emailAddress);
 
-    acc.fromConfigJSON(json.config ?? {});
+    acc.fromConfigJSON(json.configJSON ?? {});
```

Comment:
`config` is correct, within the JSON (makes no sense to have JSON within JSON). Where is `configJSON` being used within JSON specifically?

Reply:
json.config is null because in SQLMailAccount where the MailAccount is being read the field name is configJSON.

here:
https://github.com/mustang-im/mustang/blob/9e3a022aed801ce17b93e5adbedb1a2fb3d68055/app/logic/Mail/SQL/SQLMailAccount.ts#L86-L87

Reply:
I renamed the configJSON to config in SQLMailAccount and JSONMailAccount in 80ead4d1ae5de3beecdbe36466ac75b74d01b558.

Reply:
* In the SQL database, the field name is `configJSON`. But _only_ there.
* In JSON-based storage, the field name is `config`.
* The functions are called `to/fromConfigJSON()`
* In the classes, each value has its own member.